Parse-Dashboard/Authentication.js

```diff
@@ -54,14 +54,13 @@ function initialize(app, options) {
   });
 
   var cookieSessionSecret = options.cookieSessionSecret || require('crypto').randomBytes(64).toString('hex');
+  const cookieMaxAge = options.cookieMaxAge;
```

Comment:
Maybe we should call it `cookieSessionMaxAge` similar to `cookieSessionSecret`.

Reply:
changed all cookieMaxAge cookieSessionMaxAge.
Thx